lib/main_common.pm

```diff
@@ -1654,8 +1654,34 @@ sub load_wicked_tests {
         loadtest 'wicked/config_files';
     }
     elsif (check_var('WICKED', 'advanced')) {
-        loadtest 'wicked/advanced_ref' if check_var('IS_WICKED_REF', '1');
-        loadtest 'wicked/advanced_sut' if check_var('IS_WICKED_REF', '0');
+        if (check_var('IS_WICKED_REF', '1')) {
+            loadtest 'wicked/advanced/ref/bridge_interface_legacy';
+            loadtest 'wicked/advanced/ref/bridge_interface_xml';
+            loadtest 'wicked/advanced/ref/gre_tunnel_legacy';
+            loadtest 'wicked/advanced/ref/gre_tunnel_xml';
+            loadtest 'wicked/advanced/ref/ipip_tunnel_legacy';
+            loadtest 'wicked/advanced/ref/ipip_tunnel_xml';
+            loadtest 'wicked/advanced/ref/sit_tunnel_legacy';
+            loadtest 'wicked/advanced/ref/sit_tunnel_xml';
+            loadtest 'wicked/advanced/ref/tap_interface_legacy';
+            loadtest 'wicked/advanced/ref/tap_interface_xml';
+            loadtest 'wicked/advanced/ref/tun_interface_legacy';
+            loadtest 'wicked/advanced/ref/tun_interface_xml';
+        }
+        if (check_var('IS_WICKED_REF', '0')) {
```

Comment:
like @asmorodskyi mentioned but now without the number...
```
my $f = get_var('IS_WICKED_REF') ? 'ref' : 'sut';
loadtest 'wicked/advanced/' . $f . '/bridge_interface_xml';
...
```
We only need to write each testmodule once...